build.cake

```diff
@@ -0,0 +1,95 @@
+#tool "mdoc"
+
+///////////////////////////////////////////////////////////////////////////////
+// TOOLS
+///////////////////////////////////////////////////////////////////////////////
+
+var nugetPath = Context.Tools.Resolve("nuget.exe");
+var mdocPath = Context.Tools.Resolve("mdoc.exe");
+
+///////////////////////////////////////////////////////////////////////////////
+// ARGUMENTS
+///////////////////////////////////////////////////////////////////////////////
+
+var target = Argument("target", "Default");
+var configuration = Argument("configuration", "Release");
+
+///////////////////////////////////////////////////////////////////////////////
+// TASKS
+///////////////////////////////////////////////////////////////////////////////
+
+Task("Build")
+    .Does(() =>
+{
+    NuGetRestore("./F50.sln", new NuGetRestoreSettings { 
+        ToolPath = nugetPath,
+    });
+
+    MSBuild("./F50.sln", new MSBuildSettings {
+        Configuration = configuration,
+        PlatformTarget = PlatformTarget.MSIL,
+        MSBuildPlatform = MSBuildPlatform.x86,
+    });
+
+    CleanDirectories("./output/");
+    EnsureDirectoryExists("./output/");
+    CopyDirectory($"./F50/bin/{configuration}", "./output/");
+});
+
+Task("UpdateDocs")
+    .IsDependentOn("Build")
+    .Does(() =>
+{
+    // the reference folders to locate assemblies
+    var refAssemblies = "C:/Program Files (x86)/Microsoft Visual Studio/*/*/Common7/IDE/ReferenceAssemblies/Microsoft/Framework/";
```

Comment:
Do you know where these reside on the mac? Can we take a look at making it work here too?

Reply:
We can, I was just getting the Windows bits working.